scripts/src/main/resources/scripts/functions

```diff
@@ -545,7 +545,7 @@ function doArrayContainsItemWithPrefix() {
 function doMavenArchetype() {
   if [ -z "${1}" ]
   then
-    doFail "Missing arguments for doMavenArchetype ${*}"
+    doFail "Missing arguments for doMavenArchetype ${*}. Please find the documentation and examples of the parameters at https://devonfw.com/website/pages/docs/devonfw-ide-usage.asciidoc.html#java.asciidoc"
```

Comment:
`doMavenArchetype` is used for different purposes. If called from a commandlet other than `java` this change does not seem to make sense to me. Furhter, we should never link directly to our website. Sorry to say so but we have perfectly proven of the last years that our website URLs are entirely unstable. Therefore if I link to documentation, I have learned to always link to github that has a stable api as long as we do not resturcture our documentation. This worked fine over the years while or website links changed several times a year.